internal/backend/s3/config_test.go

```diff
@@ -111,3 +114,14 @@ func TestParseConfig(t *testing.T) {
 		}
 	}
 }
+
+func TestParseError(t *testing.T) {
+	const prefix = "s3: invalid format,"
+
+	for _, s := range []string{"", "/", "//", "/bucket/prefix"} {
+		_, err := ParseConfig("s3://" + s)
```

Comment:
I've changed the added prefix to `s3://` as otherwise `s3:` and `s3://` both result in the exact same `createConfig` call.